packages/docs/gatsby-node.js

```diff
@@ -13,3 +13,43 @@ exports.onCreateWebpackConfig = ({ stage, actions }) => {
     },
   });
 };
+
+exports.createPages = ({ graphql, actions }) => {
+  const { createPage } = actions;
+  const infoPageTemplate = path.resolve(`src/components/InfoPage.tsx`);
+
+  // get all pages
+  return graphql(`
+    query loadPagesQuery {
+      allMdx {
+        edges {
+          node {
+            id
+            slug
+            body
+            frontmatter {
+              title
+            }
+          }
+        }
+      }
+    }
+  `).then((result) => {
+    if (result.errors) {
+      throw result.errors;
+    }
+
+    // Create blog post pages.
+    result.data.allMdx.edges.forEach((edge) => {
+      createPage({
+        // Path for this page -- the slug with positioning markers removed
+        path: edge.node.slug.replace(/\d+_/g, ''),
```

Comment:
When we loop through these, are these _edges_ guaranteed to be in alphanumeric order? And that's why we don't need to extract that order prefix from this? And does the order in which we call `createPage` matter?

Reply:
I don't think the order that we call `createPage` matters since it is just to create routes for the site. The ordering matters more in `DocSiteSidebar`.

This string replace is to ensure that the alphanumeric ordering of the file names (which is most important in the nav creation) doesn't end up in the url path